cpp/ql/test/library-tests/dataflow/taint-tests/test_diff.expected

```diff
@@ -0,0 +1,6 @@
+| taint.cpp:35:12:35:17 | taint.cpp:41:7:41:13 | AST only |
+| taint.cpp:35:12:35:17 | taint.cpp:42:7:42:13 | AST only |
+| taint.cpp:37:22:37:27 | taint.cpp:43:7:43:13 | AST only |
+| taint.cpp:120:11:120:16 | taint.cpp:137:7:137:9 | AST only |
+| taint.cpp:127:8:127:13 | taint.cpp:130:7:130:9 | IR only |
```

Comment:
These two lines are improvements; the result on 120 is a false positive in the AST, the result on 127 is a false negative in the AST